packages/nimble-components/src/table/index.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '@tanstack/table-core';
 import { styles } from './styles';
 import { template } from './template';
-import type { TableRecord, TableFieldValue } from './types';
+import type { TableRecord } from './types';
 
```

Comment:
I'd expect the table to handle the importing of its utility components
```ts
import './cell';
import './row';
```

Reply:
I believe in #550 we established that we should use `DesignSystem.tagFor(<my-type-here>)` in the template rather than importing the sub-components within the index file as you've shown. Using `tagFor()` requires the component to be imported. From what I can see, this is also the pattern that FAST uses for importing sub-components.

Reply:
Ah, my bad, yea that makes more sense. Think I confused myself thinking the type would be elided but 6 months ago me was right lol